### DISCUSSION TITLE
请教一下关于外接显示器全屏播放的问题

### DISCUSSION THREAD

**Discussion by Tsn-09** [hooke007/mpv_PlayKit#240] [mpv 通用]
MBP外接的DELL
在笔记本桌面打开视频，然后拖到外接显示器桌面，然后双击全屏，会自动跳回到笔记本屏幕进行全屏播放

请问我要如何设置可以在外接显示器全屏播放呢？

**Comment by hooke007** [comment]
我没有条件复现mac的双屏

**Reply by hooke007** [reply]
- https://github.com/mpv-player/mpv/issues/9697

**Reply by lustm** [reply]
试了下 mpv version: 0.36x 还没有解决

**Reply by hooke007** [reply]
issue 如果 open 就表示问题存在尚未修复

**Comment by lustm** [comment]
我也是同样的问题

**Comment by lustm** [comment]
这个问题现在解决了吗

**Reply by Tsn-09** [reply]
我的解决方法是用命令行指定显示器全屏播放  mpv  1.mp4 --fs --fs-screen=1  😄

**Reply by lustm** [reply]
每次运行加命令 有点麻烦..
